src/carrier/ela_carrier.c

```diff
@@ -3699,6 +3707,8 @@ int ela_group_set_title(ElaCarrier *w, const char *groupid, const char *title)
         return -1;
     }
 
+    store_persistence_data(w);
```

Comment:
No need to store persistent data here. There is no updates for group information, here just sending a group friend.

Reply:
And I think we need to update persistent data when calling "ela_group_set_title" API.